microsoftgraph/microsoft-graph-explorer-v4
# Fix: provide account choice when logging in

## Overview

Provides account options when signing in

Fixes #488 

### Demo

N/A

### Notes

N/A

## Testing Instructions

1. Sign in and take note of the account you use
1. Sign out of that account
1. Attempt to sign in again, 
1. Notice that you will be provided with new accounts each time you try to repeat steps 1-3
